src/neuroconv/tools/yaml_conversion_specification/__init__.py

```diff
@@ -1 +1,3 @@
 from ._yaml_conversion_specification import run_conversion_from_yaml
+
+__all__ = ["run_conversion_from_yaml"]
```

Comment:
All we have to do is specify the `__all__` (again, like sklearn does)

Reply:
I thought that this only controls what `import * `

Does, can you expand on what this is solving?

Reply:
Automodule fails without pointing at the private submodule

Also not very nice to point to the private submodule since that makes it seem like a user ought to import from that private submodule